tablegen/examples/ToyOps.td

```diff
@@ -0,0 +1,454 @@
+//===- ToyOps.td - Toy dialect operation definitions -------*- tablegen -*-===//
```

Comment:
The coverage for this one test is ~50%, but perhaps you could add additional tests to cover more of the grammar?
```
$ trcover.exe ../examples/ToyOps.td
Parsing grammar C:/msys64/home/Kenne/issues/g4-4107/tablegen/Generated-CSharp/TableGen.g4
CSharp 0 ../examples/ToyOps.td success 0.0336701
Analyzing...
Analysis 0 ../examples/ToyOps.td 0.0115427
05/25-06:39:13 ~/issues/g4-4107/tablegen/Generated-CSharp
```

Output:  [cover.html.txt](https://github.com/antlr/grammars-v4/files/15442940/cover.html.txt)